### PR TITLE
feat(performance): Set API statuses on last few performance endpoints

### PR DESCRIPTION
**Pull request by shruthilayaj**

### CODE REVIEW

File: src/sentry/api/endpoints/organization_events_stats.py [hunk 1]
```diff
@@ -125,7 +125,7 @@
 @region_silo_endpoint
 class OrganizationEventsStatsEndpoint(OrganizationEventsV2EndpointBase):
     publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
+        "GET": ApiPublishStatus.EXPERIMENTAL,
```
Comment (shruthilayaj) [inline src/sentry/api/endpoints/organization_events_stats.py:128]: this is nowhere near ready to be publicized - a quarter's worth of work

File: src/sentry/api/endpoints/organization_events_trends_v2.py [hunk 2]
```diff
@@ -49,7 +49,7 @@
 @region_silo_endpoint
 class OrganizationEventsNewTrendsStatsEndpoint(OrganizationEventsV2EndpointBase):
     publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
+        "GET": ApiPublishStatus.PRIVATE,
```
Comment (shruthilayaj) [inline src/sentry/api/endpoints/organization_events_trends_v2.py:52]: only used in trends UI

File: src/sentry/api/endpoints/organization_measurements_meta.py [hunk 3]
```diff
@@ -15,7 +15,7 @@
 @region_silo_endpoint
 class OrganizationMeasurementsMeta(OrganizationEventsEndpointBase):
     publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
+        "GET": ApiPublishStatus.EXPERIMENTAL,
```
Comment (shruthilayaj) [inline src/sentry/api/endpoints/organization_measurements_meta.py:18]: custom measurements is a little up in the air, so marking it as experimental too

**Review by edwardgou-sentry — APPROVED** [review]

File: src/sentry/api/endpoints/organization_measurements_meta.py (hunk 3 above)
Comment (shruthilayaj) [inline src/sentry/api/endpoints/organization_measurements_meta.py:18]: This is meant for custom measurements (like, the metrics kind), that product area is kind of in flux right now, so marking as experimental

**Pull request merged**